docs/Table.md

```diff
@@ -72,86 +99,215 @@ const { query } = initialize({ iotsitewiseClient });
+    }
+  } | unknown
+}
 
-The text to be displays as the column header, the first row of the table
+const item: Item = {
+  rpm: {
+    $cellRef: {
+      id: 'dataStream-1',
+      resolution: 0
+    }
+  },
+  wind_speed: {
+    $cellRef: {
+      id: 'dataStream-2',
+      resolution: 0
+    }
+  },
+}
+```
+#### `$cellRef`
+$cellRef is a special keyword in Item. We use it to find the according dataStream. For the above example, 
+the item has two properties. Both of them are using `$cellRef` to map properties to data stream.
 
-Type: string
+Type: Object
 
-`rows`
+Required: No
 
-Array of content of each cell within the respective column. Rows are defined in order, the first row item corresponds to the first row on the table that doesn't represent column headers.
+$cellRef have two properties:
+- `id`
+  A string representing the id of a time series data stream, which you can get by utilizing the time series data source of choices toId helper, for example, with AWS IoT SiteWise, you can refer to an id with the following code:
+  - Type: String
+  - Required: Yes
+```typescript jsx
+import { toId } from '@iot-app-kit/source-sitewise';
+const item = [
+  {
+    wind_speed: {
+      $cellRef: {
+        id: toId({ assetId: 'engine-2', propertyId: 'temperature' }),
+        resolution: 0
+      }
+    },
+  }
+] 
+```
+- `resolution`
+The resolution, in milliseconds, at which the data should be aggregated.
+  - Type: Number  
+  - Required: Yes
+
+Other properties of an item will remain the same. 
+```typescript jsx
+// If we have defined items like below
+const items: Item[] = [{
+  rpm: {
+    $cellRef: {
+      id: 'dataStream-1',
+      resolution: 0
+    }
+  },
+  wind_speed: {
+    $cellRef: {
+      id: 'dataStream-2',
+      resolution: 0
+    }
+  },
+  other: 42
+}]
+
+// Table will receive the actual value of each $cellRef:
+[{
+  rpm: 15,
+  wind_speed: 30,
+  other: 42
+}]
+```
 
+### `columnDefinitions`
 
-Type: string or undefined
+Configures various column properties. The properties mirror [columnDefinitions](https://github.com/aws/awsui-documentation/blob/main/components/table.md#columnDefinitions)
+with the following changes:
 
-Each item must be one of the following:
+- `key`: string - Determines how each column looks up data from an `Item`.
+- (Optional)`cell` (item) => React.ReactNode* - For overriding the default [cell](link-to-default-cell-function) function.
+- (Optional)`formatter` (data:Primitive) => Primitive - Determines the display of a cell's content for formatting the data.
 
-* A string representing the id of a time series data stream, which you can get by utilizing the time series data source of choices `toId` helper, for example, with AWS IoT SiteWise, you can refer to an id with the following code:
-```
-import { toId } from '@iot-app-kit/source-sitewise';
-const column = {
-  header: 'some-header',
-  rows: [toId({ assetId: 'my-asset', propertyId: 'property-id' })]
-}
-```
+Type: Array of object
```

Comment:
Is there a type for this? also above I believe you had "Object" capitalized vs lowercase here